tests/unit/unwind_test.cc

```diff
@@ -50,7 +50,7 @@ BOOST_AUTO_TEST_CASE(test_signal_mask_is_preserved_on_unwinding) {
         // ignore
     }
 
-    // Check backtrace() unless execinfo.h is not present
+    // Check backtrace() if execinfo.h is present
```

Comment:
could you please squash this change as well?

Reply:
done it